.github/workflows/deploy.yaml

```
@@ -332,10 +332,8 @@ jobs:
           curl --proto '=https' --tlsv1.2 -sSfL https://run.linkerd.io/install | sh
       - name: Install kubectl CLI
         shell: bash
-        env:
-          K8S_VERSION: v1.30.10
         run: |
-          curl -LO https://dl.k8s.io/release/{{ env.K8S_VERSION }}/bin/linux/amd64/kubectl
+          curl -LO https://dl.k8s.io/release/v1.30.10/bin/linux/amd64/kubectl
```

I think 
```suggestion
          curl -LO https://dl.k8s.io/release/${K8S_VERSION}/bin/linux/amd64/kubectl
```
would work. I'm up for trying.

---

hahah thanks! I will update the pr